### PR TITLE
Add CSS media queries responsiveness

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Add Media query breakpoints
Milestone 2: Add CSS for desktop responsiveness.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -304,6 +335,149 @@ h2 {
...
+    place-self: center;
+    margin-bottom: 100px;
+  }
+
+  #contact-btn {
+    place-self: center;
+  }
+}
+
+@media (min-width: 900px) {
```
Comment (AshakaE) [inline styles.css]: Too low

- [ ] Please pick a value between `992 and 1100px`  to target desktop screens 🙇‍♂️

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Hector096  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
### _Check the comments under the review._

- [ ] Please fix the image distortion

Template |
------------ | 
image

Your Copy |
------------- |
 image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Hector096  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!



#  Required Changes ♻️ 🚧

- [ ] Please work on the distorted images in the projects section, 
_see previous review for reference_ 😄 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Hector096) [reply styles.css]: can you please explain a little more about image distortion of images because it's not happening on my browser?

**Review by AshakaE — APPROVED** [review]
### Hi @Hector096  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 🚀 🚀 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**